taxi/views.py

```diff
@@ -87,3 +89,34 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+
+class CarAssignDriverView(LoginRequiredMixin, generic.View):
+    @staticmethod
+    def post(request: HttpRequest, pk: int) -> HttpResponseRedirect:
+        car = get_object_or_404(Car, id=pk)
+        car.drivers.add(request.user)
+        return redirect("taxi:car-detail", pk=pk)
+
+
+class CarRemoveDriverView(LoginRequiredMixin, generic.View):
+    @staticmethod
+    def post(request: HttpRequest, pk: int) -> HttpResponseRedirect:
+        car = get_object_or_404(Car, id=pk)
+        car.drivers.remove(request.user)
```

Comment:
Ensure that the user is a driver before removing them from the car's drivers. This can be done by checking the user's role or permissions.